charms/mlflow-server/metadata.yaml

```diff
@@ -5,13 +5,10 @@ description: |
   MLflow
 summary: |
   A charm which provides a Kubernetes installation of the MLflow (https://mlflow.org/) server.
-series: [kubernetes]
```

Comment:
Should we leave `series` as per our discussion with @knkski ?

Reply:
agreed.  updating